File: Beginner/1020 - Age in Days/1020.c
#include <stdio.h>
#include <math.h>

int main()
{

    int dias, anos, meses;

    scanf("%i", &dias);

    anos = dias / 365;
    dias %= 365;
    meses = dias / 30;
    dias %= 30;

    printf("%i ano(s)\n", anos);
    printf("%i mes(es)\n", meses);
    printf("%i dia(s)\n", dias);

    return 0;
}

File: Beginner/1020 - Age in Days/1020.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        int day = int.Parse(Console.ReadLine());

        int year = day / 365;
        day %= 365;
        int month = day / 30;
        day %= 30;

        Console.WriteLine($"{year} ano(s)");
        Console.WriteLine($"{month} mes(es)");
        Console.WriteLine($"{day} dia(s)");
    }
}